applications/asset_tracker/src/ui/nmos.c

```diff
@@ -184,6 +194,17 @@ int ui_nmos_init(void)
 		return -ENODEV;
 	}
 
+	for (size_t i = 0; i < ARRAY_SIZE(nmos_pins); i++) {
+		pwm_pin_set_usec(
+			pwm_dev,
```

Comment:
We typically don't start the argument list in a new line (unless we're constrained by the char limit).  See line 95.

Reply:
Should be fixed now